tests/integration/calendar/test_calendar_props.py

```diff
@@ -0,0 +1,110 @@
+import itertools
+import pytest
+
+import dash_core_components as dcc
+import dash_html_components as html
+import dash
+from dash.dependencies import Input, Output
+import dash.testing.wait as wait
+
+DAY_SELECTOR = 'div[data-visible="true"] td.CalendarDay'
+
+
+@pytest.mark.DCC594
```

Comment:
this is an idea I have in mind to associate the test case with issue number, could be helpful for test selection by associating the test case with the issue

Reply:
@byronz we can keep this if it's useful, but we should do something to clear the warning it currently gives:

> /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/mark/structures.py:324: PytestUnknownMarkWarning:
> Unknown pytest.mark.DCC594 - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html